app/static/about.css

```diff
@@ -0,0 +1,56 @@
+body {
+    margin: 0;
+    padding: 0;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #F7F7F7;
+}
+
+.moving-words {
+    font-family: "Assistant", "Ariel", sans-serif;
+    text-transform: uppercase;
+    font-size: 6em;
+    letter-spacing: 0.1em;
+    overflow: hidden;
+    background: linear-gradient(90deg, #000, #fff, #000);
+    background-repeat: no-repeat;
+    background-size: 80%;
+    animation: animate 4s linear infinite;
+    -webkit-background-clip: text;
+    background-clip: text;
+    -webkit-text-fill-color: rgba(255, 255, 255, 0);
+}
+
+@keyframes animate {
+    0% {
+        background-position: -500%;
+    }
+    100% {
+        background-position: 500%;
+    }
+}
+
+
+/* credit for the animation goes to FrankieDoodie */
+
+.infographic {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+}
+
+.abouttext {
+    font-size: 1.2em;
+    text-align: justify;
+}
+
+a {
```

Comment:
this is already set the the global.css, please connect it to your pages.

Reply:
added.